src/dmd/e2ir.d

```diff
@@ -89,17 +89,19 @@ void* mem_malloc2(uint);
  */
 bool ISREF(Declaration var)
 {
-    return (config.exe == EX_WIN64 && var.isParameter() &&
-            (var.type.size(Loc.initial) > REGSIZE || var.storage_class & STC.lazy_))
+    return (var.isParameter() &&
+            ((!global.params.isWindows && var.type.isTypeStruct() && !var.type.isTypeStruct().sym.isPOD()) ||
```

Comment:
Having things like `!global.params.isWindows` is a future bug lying in wait. It assumes that all future systems supported by the compiler that are not Windows will conform to this ABI.
It's far batter to enumerate which targets it applies to, which is what `config.exe` is for, as it is a bit mask.